### DISCUSSION TITLE
Camera Switching Glitch For Uploading Documents

### DISCUSSION THREAD

**Discussion by ahmadrazacdx** [community/community#144740] [GitHub Education]
### Hi everyone,

I'm having an issue when I proceed my application. Under the section of uploading my documents when I grant permission to camera, it always opens front facing camera upon switching to back. It still remains front facing camera opened. So it makes impossible to upload documents.

**Comment by Serkom2000** [comment]
Same, I found a small solution, On your phone, turn off camera permission and then turn it on (in Chrome, the settings icon is at the top left near the www.github.com) Then change the camera to the back one and it will change
